v2/sourcedb-to-spanner/src/main/java/com/google/cloud/teleport/v2/options/SourceDbToSpannerOptions.java

```diff
@@ -263,4 +263,15 @@ public interface SourceDbToSpannerOptions extends CommonTemplateOptions {
   Boolean getInsertOnlyModeForSpannerMutations();
 
   void setInsertOnlyModeForSpannerMutations(Boolean value);
+
+  @TemplateParameter.Text(
+      order = 22,
+      optional = true,
+      description = "BatchSize for Spanner Mutation.",
+      helpText =
+          "BatchSize in bytes for Spanner Mutations. if set less than 0, default of Apache Beam's SpannerIO is used, which is 1MB. Set this to 0 or 10, to disable batching mutations.")
+  @Default.Long(-1)
+  Long getBatchSizeForSpannerMutations();
```

Comment:
We should keep an internal TODO to track the repercussions of switching to unbatched. If the throughput is same as Cassandra MAx rread throughput, maybe its worth making it a default config for all sources.

Reply:
Added and shared over chat.